packages/admin-ui/src/filters/MultipleItemsLabel.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { tag } from '@vtex/admin-ui-react'
+
+export const MultipleItemsLabel = (props: { appliedItems: any[] }) => {
+  const { appliedItems } = props
+
+  const firstSelectedItemLabel =
+    appliedItems.length > 1
+      ? `${appliedItems[0]?.label},`
+      : `${appliedItems[0]?.label}` // no comma in this case
+
+  const remainingSelectedItemsCount =
+    appliedItems.length > 1 && `+${appliedItems.length - 1}`
```

Comment:
I would rather use ternary to keep the same return type, avoiding to return `false` for this expression:
```suggestion
  const remainingSelectedItemsCount =
    appliedItems.length > 1 ? +${appliedItems.length - 1}` : ''
```